src/fides/api/app_setup.py

```diff
@@ -171,8 +171,9 @@ async def run_database_startup(app: FastAPI) -> None:
     if CONFIG.database.automigrate:
         try:
             configure_db(CONFIG.database.sync_database_uri)
-            with get_autoclose_db_session() as session:
-                seed_db(session)
+            if not CONFIG.test_mode:
+                with get_autoclose_db_session() as session:
+                    seed_db(session)
```

Comment:
This is something I missed during the test cleanup a while back. We don't want to seed the data in test mode. We want the tests to run from a clean start.